tiledb/bioimg/converters/base.py

```diff
@@ -410,6 +410,26 @@ def to_tiledb(
         metadata = {}
         original_metadata = {}
 
+        # Checks group validity
+        overwrite = False
```

Comment:
Should `overwrite` be an arg that defaults to `False`? That would give us flexibility to overwrite every image. Although uncommon, may be times when we need to force overwrite. Could then make that arg accessible from `tiledb.cloud.bioimage.ingest` s well.

Reply:
The caveat would be of course that if an image is invalid it will still be overwritten even if `overwrite=False` set in the caller

Reply:
Yeap the direction of this patch is to mitigate retrial issues and corrupted leftovers, that the user fails to recognize. If users really want to overwrite/re-ingest an image on purpose they can always do it by changing the destination path and manually deleting the previous record.